Source/SpeechToTextV1/SpeechToText.swift

```diff
@@ -325,13 +334,17 @@ public class SpeechToText {
        parameter. To determine whether a language model supports speaker labels, use the **Get models** method and check
        that the attribute `speaker_labels` is set to `true`. See [Speaker
        labels](https://console.bluemix.net/docs/services/speech-to-text/output.html#speaker_labels).
+     - parameter customizationID: **Deprecated.** Use the `language_customization_id` parameter to specify the
+       customization ID (GUID) of a custom language model that is to be used with the recognition request. Do not
+       specify both parameters with a request.
```

Comment:
Shouldn't this parameter appear after `model`?  It does in the parameter list.  I think we should probably document the parameters in the same order as they appear in the parameter list.